src/controllers/connection/newConnection.ts

```diff
@@ -195,30 +196,52 @@ export class NewConnectionController extends HTMLController {
     return this.newInviteSuccessHtml(formStage, company, body.email)
   }
 
+  /**
+   * render pin code submission form
+   * @param companyNumber - for retrieving a connection from a db
+   * @param pin - a pin code
+   * @returns
+   */
+  @SuccessResponse(200)
+  @Get('/pin-submission')
```

Comment:
Can I suggest we move these under a path that includes the connection uuid? This will make this much easier to integrate with the actual credential proposal. So for example this would move to the `connection` controller and be under the path:

`/connection/{uuid}/pin-submission`

Reply:
This should be updated, currently left in new/ just because of the form, but will move it now to the correct patch e.g. `/connection/{id}/pin-submission`